src/components/interface/SectionHeader.jsx

```diff
@@ -29,7 +28,7 @@ const SectionHeader = ({
 );
 
 SectionHeader.propTypes = {
-  classes: PropTypes.string.isRequired,
+  classes: PropTypes.shape().isRequired,
```

Comment:
assurance on shape maybe?

Reply:
@YangStephen  would like to sync with you on this, not sure how to add assurance on the classes object